### DISCUSSION TITLE
官媒用AI假主播 IT專家憂中共更多造假

### DISCUSSION THREAD

**Discussion by 20010906** [1101-GS4719-C/Python-Community-GS4719-C#332] [General]
前陣子，我分享了關於AI現在能高速生產並撰寫新聞的報導，但AI的能力不只侷限於此，中國研發出AI擬真的新聞主播，且在測試一段時間後，也絲毫沒有破綻，可以讓我們知道AI快速的進步以及便利，但也不免會擔心日後是否遭有心人士濫用。
[AI假主播](https://www-epochtimes-com.cdn.ampproject.org/v/s/www.epochtimes.com/b5/22/1/9/n13491484.htm/amp?amp_js_v=a6&amp_gsa=1&usqp=mq331AQKKAFQArABIIACAw%3D%3D#aoh=16418828688170&referrer=https%3A%2F%2Fwww.google.com&amp_tf=%E4%BE%86%E6%BA%90%EF%BC%9A%251%24s&ampshare=https%3A%2F%2Fwww.epochtimes.com%2Fb5%2F22%2F1%2F9%2Fn13491484.htm)

**Comment by chun1203** [comment]
現在的AI既可以寫新聞又可以充當記者在節目上播報 這樣記者是不是要沒工作了QQ

**Comment by 107602505** [comment]
雖然AI可以提高效率且無論在計算法、性能，還是學習能力上，都已經遠遠超越人類，但是人工智慧最大的問題在於，它沒有倫理道德還是必須經由人類操控，若連操作都能由AI取代的話那未來人類就真的要沒有工作了QQ。

**Reply by enoch-lightning** [reply]
我不覺得AI的學習能力遠遠超過人類。因為我覺得AI的學習能力其實是在人類的掌握之中。是人類設計讓AI能學得好，AI才能學得好。舉深度學習來說，若評估指標是「正確率(accuracy)」，我可以架出一個training accuracy, validation accuracy與testing accuracy都很高的模型，當然可能要使用一些技巧，如「遷移學習(transfer learning)。」但，我也可以架出一個效果很差，或是一下效果好，一下效果不好，也就是不穩定的模型。

**Comment by YuHengChen54** [comment]
只能說AI現在真的越做越好了，真的快要跟人類一樣了
